types/wlr_screenshooter.c

```diff
@@ -41,27 +30,32 @@ static void handle_screenshot_resource_destroy(
 	}
 }
 
-static void output_frame_notify(struct wl_listener *listener, void *_data) {
+static void output_handle_frame(struct wl_listener *listener, void *_data) {
```

Comment:
I think we've settled on something like `handle_output_frame` for callbacks but there's no accepted standard yet.

Reply:
I prefer `<type>_handle_<event>`.